ui/imports/shared/status/StatusEmojiPopup.qml

```diff
@@ -15,6 +15,7 @@ import shared.controls 1.0
 
 Popup {
     id: popup
+    objectName: "emojiPopup"
```

Comment:
I don't think this name is used, but I guess we can keep it

Reply:
missed it, sorry :)